src/components/shared/cellRenderers/isSimulatedRenderer/isSimulatedRenderer.js

```diff
@@ -2,15 +2,11 @@
 
 import React from "react";
 
-import { Svg } from 'components/shared/svg/svg';
-import { svgs } from 'utilities';
-
 import '../cellRenderer.css';
 
 export const IsSimulatedRenderer = ({ value, context: { t } }) => (
   <div className="pcs-renderer-cell highlight">
-    <Svg path={ value ? svgs.simulatedDevice : svgs.physicalDevice } className="pcs-renderer-icon" />
-    <div className="pcs-renderer-text uppercase">
+    <div className="pcs-renderer-text">
```

Comment:
Why are we removing these icons?

Reply:
@CaitlinEsworthy can you comment on the icon removal?